packages/lodestar/src/cli/commands/beacon.ts

```diff
@@ -82,7 +83,15 @@ export class BeaconNodeCommand implements CliCommand {
         ),
         beacon: new BeaconApi(
           {},
-          {config, chain: this.node.chain, db: this.node.db}
+          {
+            config,
+            logger: new WinstonLogger(),
+            sync: this.node.sync,
+            eth1: this.node.eth1,
+            opPool: this.node.opPool,
+            chain: this.node.chain,
+            db: this.node.db
+          }
```

Comment:
I realize that this is just updating what was already here, but this is where we would be using the beacon node's Beacon/ValidatorApis.